new/LIT7083QeneTadewewotu.xml

```diff
@@ -0,0 +1,90 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7083QeneTadewewotu" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ተጼውዎቱ፡ ለ<choice><sic>ስ</sic><corr resp="JK">ሰ</corr></choice>ብእ፡ እመ፡ ከመ፡ ጽዮን፡ ኮነ፡ ምህርካ፡ ጽኑእ፡ እም፡ ተመነነ፤፤</title>
```

Comment:
I would not expect ፤፤ in the title (or the editorial mark up) - this belongs to the edition

Reply:
Ok! I will use a standardized form in the title